integ-test/src/test/java/org/opensearch/sql/sql/SQLCorrectnessIT.java

```diff
@@ -23,6 +28,9 @@ public class SQLCorrectnessIT extends CorrectnessTestBase {
   @Override
   protected void init() throws Exception {
     super.init();
+    updateClusterSettings(
```

Comment:
is this needed?  I feel like this is unnecessary for the tests to pass.

Reply:
It it `true` by default, but previous test may set it to `false`.